helm/conjur-config-cluster-prep/values.yaml

```diff
@@ -31,19 +31,19 @@ authnK8s:
     # The name of the authenticator ClusterRole to use or create.
```

Comment:
Should authn-k8s-configmap above L#26 be changed also?

Reply:
Yes, indeed. Good catch!